### ISSUE TITLE
Image Folder Widget won't download files if they are changed for different files with the same name

### ISSUE THREAD

**Issue by PatrickA-RiseVision** [Rise-Vision/rise-launcher-electron#771]
- Operating System Version: Ubuntu 14.04, Ubuntu 16.04
- Player Version: RisePlayerElectron 2018.10.17.17.05 Licensed
- Viewer Version: 2-01-201810182053

##### What is the expected behavior?
When a folder is selected in the Image Widget, images in the folder will run as a slideshow.

##### What is the actual behavior?
When a folder is selected in the Image Widget, images sometimes will not download to play after the files are changed for files with the same names.

##### What steps can be taken to replicate the problem? 

1. Create a folder in Rise Storage and upload a series of images to it.
1. Create a presentation that uses the Image Widget to play the folder you've created.
1. Set the widget to Play Until Done
1. Delete the images from Rise Storage.
1. Upload a series of images with the same names to the folder.

When the issue is triggered, the message indicating files are downloading will appear instead of the image widget.  Since it's PUD, it won't go away.

**Comment by tejohnso** [comment]
@PatrickA-RiseVision a clarification request:

After step 3 we can confirm that the presentation should be playing as expected, correct?

For step 4 does it matter if the deletion happens while the presentation is running, or should the display be offline?

When you say "Delete" in step 4 do you mean move to trash then permanently delete from trash, or is a simple move to trash sufficient?

**Comment by PatrickA-RiseVision** [comment]
Thanks @tejohnso 

I wish I knew for sure; we haven't had any luck replicating this which is why the defect log is so vague.  @andrecardoso and @rudibravo have been looking into it, but found nothing in logs either.  I'll answer as best as I can:

- After step 3, the presentation should be playing correctly.

- For step 4, I am of the impression the displays are offline when the deletion happens, but the customer affected by the issue says their displays are never offline when they update their content.  The user we've been working with was able to replicate it 1 time on a display that had been offline when files were updated, and the issue happened immediately, so I think it happens if the display is offline when the folder's contents are modified.
 
- I'm actually not sure.  Let me see if I can obtain clarification from the user.  My assumption was that they were deleting it, but I did not try uploading on top of the files nor permanently deleting them.  I will test on my display.
